src/utilities/HttpClient.ts

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+
+const instance = axios.create({
+  baseURL: 'https://mate.academy/students-api',
+});
+
+export const HttpClient = {
```

Comment:
```suggestion
export const httpClient = {
```
use camelCase if it's not React component/type/class etc.